### ISSUE TITLE
@startuml or @startgantt stops toc generator from functioning. 

### ISSUE THREAD

**Issue by GeoffMurray** [huntertran/markdown-toc#60]
I am using plantuml code blocks, like so...

\`\`\`plantuml
@startuml
start
:Init Phase;
:Transfer Phase;
note right
  long running activity,
  process requires signal to proceed
end note
:Termination Phase;
stop
@enduml
\`\`\`

Or, this:

\`\`\`plantuml
@startgantt
skinparam monochrome true
[Task1] on {Alice} lasts 8 days
then [Task2] on {Bob} lasts 4 days at 50%
then [Task3] on {Carol} lasts 2 days at 25%
@endgantt
\`\`\`

Either of these stop the TOC generator from functioning. In the first example, I can workaround by removing the @startuml and @enduml markers. However, in the second example, I cannot, because that's a critical part of the syntax. 

It's as though the TOC generator is not really treating the code block as fully code. It seems something about the "@" syntax is throwing off the processing. 

I'm not sure if there is any sort of error log I might be able to find to help narrow down the problem.

**Comment by vamsi-juvvi** [comment]
I have been doing plantuml heavily and have notice this too. For a while I switched to using VSCode's outline feature as a replacement but decided to try my hand at actually debugging my first vsCode extension. Code is straightforward and well structured so was fairly quick to locate the issue.

Will send a PR for the simplest, clean fix I can come up with in a bit.

**Comment by vamsi-juvvi** [comment]
@huntertran  Created PR: https://github.com/huntertran/markdown-toc/pull/62. Pls let me know if you want me to change anything.

@GeoffMurray and others. To use the changes locally while the PR is evaluated for merge, do the following

```bash
> git clone https://github.com/vamsi-juvvi/markdown-toc.git
> cd markdown-toc
> npm install -g @vscode/vsce
> vsce package
```

`vsce package` should result in generation of `auto-markdown-toc-3.0.13.vsix`.

Follow instructions from https://stackoverflow.com/questions/42017617/how-to-install-vs-code-extension-manually to upgrade to the locally built auto-markdown-toc-3.0.13.vsix package
 - These are older instructions: the `Extensions` icon in the left bar has changed but the rest of the instructions still apply in 2023.

**Comment by huntertran** [comment]
Thank @vamsi-juvvi for the pull request
The fix was published in 3.0.13
